frontend/src/components/BetaAlert.tsx

```diff
@@ -1,18 +1,28 @@
 import { useTranslations } from "next-intl";
+import { GridContainer, Alert as USWDSAlert } from "@trussworks/react-uswds";
 
-import FullWidthAlert from "./FullWidthAlert";
+type Props = {
+  containerClasses?: string;
+};
 
-const BetaAlert = () => {
+const BetaAlert = ({ containerClasses }: Props) => {
   const t = useTranslations("Beta_alert");
   const alert = t.rich("alert", {
     LinkToGrants: (content) => <a href="https://www.grants.gov">{content}</a>,
   });
 
   return (
-    <div data-testid="beta-alert">
-      <FullWidthAlert type="info" heading={t("alert_title")}>
-        {alert}
-      </FullWidthAlert>
```

Comment:
No longer need the FullWidthAlert for the BetaAlert. We might deprecate this component in the future. But we can leave it for now in case we need it for other purposes.